debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -832,11 +865,8 @@ public String getConnectionUrl() {
 
             @Override
             public StreamingAdapter getInstance(OracleConnectorConfig connectorConfig) {
-                return Instantiator.getInstanceWithProvidedConstructorType(
-                        "io.debezium.connector.oracle.xstream.XStreamAdapter",
-                        StreamingAdapter.class::getClassLoader,
-                        OracleConnectorConfig.class,
-                        connectorConfig);
+                return Instantiator.getInstanceWithProvidedConstructorType("io.debezium.connector.oracle.xstream.XStreamAdapter", StreamingAdapter.class::getClassLoader,
```

Comment:
It looks like there was some reformatting here that doesn't involve any code changes.  I'd prefer to keep the prior formats unless there's a reason to change this and the few other changes in this file that follow below with the various adapters and processors.

Reply:
I thought this was because of `process-sources` maybe? @mdrillin ?

Reply:
I run the formatter quite often on Oracle as you can imagine and I haven't seen these changes pop up in any of my Oracle PRs, hence why I was surprised to see them :)

Reply:
I reverted to the original formatting.  (I thought the build process was re-formatting it, so I merged it previously.  But I think it was just a side effect of the build failing somewhere else).